css-positioning/absolute.css

```diff
@@ -0,0 +1,35 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  right: 30px;
+  top: 10px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: relative;
+  top: 175px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
```

Comment:
please use the position: absolute property for the overlay as well as the top, right, bottom, and left properties to make sure the gradient goes over the image!

Reply:
<img width="1502" alt="Screenshot 2023-01-27 at 10 10 11 AM" src="https://user-images.githubusercontent.com/122064415/215162385-7bbcd840-23b5-4432-9ad2-8d7d76b96dea.png">